ivy/functional/ivy/device.py

```diff
@@ -305,8 +308,30 @@ def print_all_ivy_arrays_on_dev(
     ----------
     device
         The device on which to print the arrays
+    
     attr_only
         Whether or not to only print the `shape` and `dtype` attributes of the array
+    
+    Examples
+    --------
+
+    With attr_only = True
+
+    >>> x = ivy.array([1,0,2], [3,2])
```

Comment:
Hello @WilliamHirst , thank you for your effort! The array should have an identical number of elements per dimension. So , you can change x to be = `ivy.array([[1,0,2],[3,2,1]])` instead of `ivy.array([1,0,2], [3,2])`. You should then update the function call's results accordingly.

Reply:
Oh yes, I missed that! Thank you for the feedback. It is now fixed.